src/test/java/com/linkedin/kmf/services/MultiClusterTopicManagementServiceTest.java

```diff
@@ -60,9 +60,11 @@ private void startTest() {
     nodeSet.add(new Node(3, "host-3", 2134));
     nodeSet.add(new Node(4, "host-4", 2135));
     nodeSet.add(new Node(5, "host-5", 2136));
-    nodeSet.add(new Node(6, "host-5", 2136));
-    nodeSet.add(new Node(7, "host-5", 2136));
-    nodeSet.add(new Node(8, "host-5", 2136));
+    nodeSet.add(new Node(6, "host-5", 2137));
```

Comment:
Consider a negative test that throws`org.apache.kafka.common.errors.InvalidReplicaAssignmentException`

Reply:
will address your concerns in a sep. PR.